### PR TITLE
DB/NPC:  Stable Masters

### LINKED ISSUE #14720: DB/NPC: Fixing some Stable Masters
Fix all gossips for stable masters and add gossip_option required to recover lost pets.
``` sql
UPDATE `creature_template` SET `npcflag`=4194305 WHERE `entry`=9988;
UPDATE `conditions` SET `ConditionValue1` =4 WHERE `SourceGroup`=9576 AND `SourceTypeOrReferenceId`=15;
UPDATE `creature_template` SET `npcflag`=4194305 WHERE  `npcflag`=4194304 ;
UPDATE `creature_template` SET `npcflag`=4194307 WHERE  `npcflag`=4194306 ;
UPDATE `creature_template` SET `gossip_menu_id`=9821 WHERE `entry` IN (9976,9978,9979,9982,9985,9986,10048,10049,10051,10052,10057,10058,11105,13616,15131,16094,16586,16656,16824,17896,18984,19018,21336,21517,22469,9896,19325,22551,22577,23392,24905,24974,25037,25519,29251,30155,31991,32105,35290);

DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` IN (14) AND `SourceGroup`=9821;
INSERT INTO `conditions` (`SourceTypeOrReferenceId`,`SourceGroup`,`SourceEntry`,`ElseGroup`,`ConditionTypeOrReference`,`ConditionValue1`,`ConditionValue2`,`ConditionValue3`,`ErrorTextId`,`ScriptName`,`Comment`, `NegativeCondition`) VALUES
(14, 9821, 13557, 0, 15, 4, 0, 0, 0, '', 'Show gossip text if player is a hunter', 0),
(14, 9821, 13584, 0, 15, 4, 0, 0, 0, '', 'Show gossip text if player is not a hunter', 1);

UPDATE `gossip_menu_option` SET `option_id`=1, `npc_option_npcflag`=1, `action_menu_id`=9820 WHERE `menu_id`=9821 AND `id`=1;

DELETE FROM `gossip_menu` WHERE `entry`=11971 AND `text_id`=16788;
INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES (11971,16788);

DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` IN (14) AND `SourceGroup`=11971;
INSERT INTO `conditions` (`SourceTypeOrReferenceId`,`SourceGroup`,`SourceEntry`,`ElseGroup`,`ConditionTypeOrReference`,`ConditionValue1`,`ConditionValue2`,`ConditionValue3`,`ErrorTextId`,`ScriptName`,`Comment`, `NegativeCondition`) VALUES
(14, 11971, 16788, 0, 15, 4, 0, 0, 0, '', 'Show gossip text if player is a hunter', 0),
(14, 11971, 16789, 0, 15, 4, 0, 0, 0, '', 'Show gossip text if player is not a hunter', 1);

DELETE FROM `npc_text` WHERE id=16788;
INSERT INTO `npc_text` (`ID`, `text0_0`, `BroadcastTextID0`, `Probability0`) VALUES 
(16788, "Hey der, mon.$B$BDid ya want ta be stablin' yer pets, or were ya lookin' for a lost companion?", 45395, 1);

DELETE FROM `gossip_menu_option` WHERE `menu_id` = 11971;
INSERT INTO `gossip_menu_option` (`menu_id`,`id`,`option_icon`,`option_text`,`OptionBroadcastTextID`,`option_id`,`npc_option_npcflag`,`action_menu_id`,`action_poi_id`,`box_coded`,`box_money`,`box_text`,`BoxBroadcastTextID`) VALUES
(11971,0,0,'I''d like to stable my pet here.',30181,14,4194304,0,0,0,0,'',0),
(11971,1,1,'I''m looking for a lost companion.',56613,1,1,9820,0,0,0,'',0);

DELETE FROM `gossip_menu` WHERE `entry`=9864 AND `text_id`=13662;
INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES (9864,13662);

UPDATE `npc_text` SET `text0_0`="", `text0_1`="Well met, Battle-Sister. I can help stable your pets or assist you in recovering lost companions.", `BroadcastTextID0`=30667 WHERE `ID`=13662;

DELETE FROM `gossip_menu_option` WHERE `menu_id` = 9864;
INSERT INTO `gossip_menu_option` (`menu_id`,`id`,`option_icon`,`option_text`,`OptionBroadcastTextID`,`option_id`,`npc_option_npcflag`,`action_menu_id`,`action_poi_id`,`box_coded`,`box_money`,`box_text`,`BoxBroadcastTextID`) VALUES
(9864,0,0,'I''d like to stable my pet here.',30181,14,4194304,0,0,0,0,'',0),
(9864,1,1,'I''m looking for a lost companion.',56613,1,1,9820,0,0,0,'',0);

DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` IN (14) AND `SourceGroup`=9864;
INSERT INTO `conditions` (`SourceTypeOrReferenceId`,`SourceGroup`,`SourceEntry`,`ElseGroup`,`ConditionTypeOrReference`,`ConditionValue1`,`ConditionValue2`,`ConditionValue3`,`ErrorTextId`,`ScriptName`,`Comment`, `NegativeCondition`) VALUES
(14, 9864, 13662, 0, 15, 4, 0, 0, 0, '', 'Show gossip text if player is a hunter', 0),
(14, 9864, 13661, 0, 15, 4, 0, 0, 0, '', 'Show gossip text if player is not a hunter', 1);

DELETE FROM `gossip_menu_option` WHERE `menu_id` = 9212 AND `id`=1;
INSERT INTO `gossip_menu_option` (`menu_id`,`id`,`option_icon`,`option_text`,`OptionBroadcastTextID`,`option_id`,`npc_option_npcflag`,`action_menu_id`,`action_poi_id`,`box_coded`,`box_money`,`box_text`,`BoxBroadcastTextID`) VALUES
(9212,1,0,'I''d like to stable my pet here.',30181,14,4194304,0,0,0,0,'',0);

DELETE FROM `gossip_menu` WHERE `entry`=9820 AND `text_id`=13584;
INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES (9820,13584);

DELETE FROM `gossip_menu_option` WHERE `menu_id` = 9820;
INSERT INTO `gossip_menu_option` (`menu_id`,`id`,`option_icon`,`option_text`,`OptionBroadcastTextID`,`option_id`,`npc_option_npcflag`,`action_menu_id`,`action_poi_id`,`box_coded`,`box_money`,`box_text`,`BoxBroadcastTextID`) VALUES
(9820,0,0,'I''ve lost Miniwing.',30182,1,1,0,0,0,0,'',0),
(9820,1,0,'I''ve lost my Jubling.',30183,1,1,0,0,0,0,'',0),
(9820,2,0,'I''ve lost my sprite darter hatchling.',30192,1,1,0,0,0,0,'',0),
(9820,3,0,'I''ve lost my worg pup.',30196,1,1,0,0,0,0,'',0),
(9820,4,0,'I''ve lost my Smolderweb hatchling.',30197,1,1,0,0,0,0,'',0),
(9820,5,0,'I''ve lost my Prairie Chicken.',30217,1,1,0,0,0,0,'',0),
(9820,6,0,'I''ve lost my wolpertinger.',30224,1,1,0,0,0,0,'',0);

DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` IN (15) AND `SourceGroup`=9820;
INSERT INTO `conditions` (`SourceTypeOrReferenceId`,`SourceGroup`,`SourceEntry`,`ElseGroup`,`ConditionTypeOrReference`,`ConditionValue1`,`ConditionValue2`,`ConditionValue3`,`ErrorTextId`,`ScriptName`,`Comment`, `NegativeCondition`) VALUES
(15, 9820, 0, 0, 8, 10898, 0, 0, 0, '', 'Show the gossip option if the player rewarded the quest', 0),
(15, 9820, 0, 0, 2, 31760, 1, 1, 0, '', 'Show the gossip option if the player Didn''t have the item', 1),
(15, 9820, 0, 0, 25, 39181, 0, 0, 0, '', 'Show the gossip option if the player Didn''t lear the spell', 1),
(15, 9820, 1, 0, 8, 7946, 0, 0, 0, '', 'Show the gossip option if the player rewarded the quest', 0),
(15, 9820, 1, 0, 2, 19450, 1, 1, 0, '', 'Show the gossip option if the player Didn''t have the item', 1),
(15, 9820, 1, 0, 2, 19462, 1, 1, 0, '', 'Show the gossip option if the player Didn''t have the item', 1),
(15, 9820, 1, 0, 25, 23811, 0, 0, 0, '', 'Show the gossip option if the player Didn''t lear the spell', 1),
(15, 9820, 2, 0, 8, 4298, 0, 0, 0, '', 'Show the gossip option if the player rewarded the quest', 0),
(15, 9820, 2, 0, 2, 11474, 1, 1, 0, '', 'Show the gossip option if the player Didn''t have the item', 1),
(15, 9820, 2, 0, 25, 15067, 0, 0, 0, '', 'Show the gossip option if the player Didn''t lear the spell', 1),
(15, 9820, 3, 0, 8, 4729, 0, 0, 0, '', 'Show the gossip option if the player rewarded the quest', 0),
(15, 9820, 3, 0, 2, 12264, 1, 1, 0, '', 'Show the gossip option if the player Didn''t have the item', 1),
(15, 9820, 3, 0, 25, 15999, 0, 0, 0, '', 'Show the gossip option if the player Didn''t lear the spell', 1),
(15, 9820, 4, 0, 8, 4862, 0, 0, 0, '', 'Show the gossip option if the player rewarded the quest', 0),
(15, 9820, 4, 0, 2, 12529, 1, 1, 0, '', 'Show the gossip option if the player Didn''t have the item', 1),
(15, 9820, 4, 0, 25, 16450, 0, 0, 0, '', 'Show the gossip option if the player Didn''t lear the spell', 1),
(15, 9820, 5, 0, 8, 3861, 0, 0, 0, '', 'Show the gossip option if the player rewarded the quest', 0),
(15, 9820, 5, 0, 2, 11110, 1, 1, 0, '', 'Show the gossip option if the player Didn''t have the item', 1),
(15, 9820, 5, 0, 25, 13548, 0, 0, 0, '', 'Show the gossip option if the player Didn''t lear the spell', 1),   
(15, 9820, 5, 1, 8, 13840, 0, 0, 0, '', 'Show the gossip option if the player rewarded the quest', 0),
(15, 9820, 5, 1, 2, 11110, 1, 1, 0, '', 'Show the gossip option if the player Didn''t have the item', 1),
(15, 9820, 5, 1, 25, 13548, 0, 0, 0, '', 'Show the gossip option if the player Didn''t lear the spell', 1),
(15, 9820, 6, 0, 8, 11117, 0, 0, 0, '', 'Show the gossip option if the player rewarded the quest', 0),
(15, 9820, 6, 0, 2, 32233, 1, 1, 0, '', 'Show the gossip option if the player Didn''t have the item', 1),
(15, 9820, 6, 0, 25, 39709, 0, 0, 0, '', 'Show the gossip option if the player Didn''t lear the spell', 1),
(15, 9820, 6, 1, 8, 11431, 0, 0, 0, '', 'Show the gossip option if the player rewarded the quest', 0),
(15, 9820, 6, 1, 2, 32233, 1, 1, 0, '', 'Show the gossip option if the player Didn''t have the item', 1),
(15, 9820, 6, 1, 25, 39709, 0, 0, 0, '', 'Show the gossip option if the player Didn''t lear the spell', 1);

UPDATE `creature_template` SET `ScriptName`="npc_stable_master" WHERE `entry` IN (16764, 9986, 9987, 9989, 10045, 10046, 10047, 10048, 10049, 10050, 10051, 10053, 10054, 10055, 10056, 10057, 10058, 10059, 10060, 10061, 10062, 10063, 10085, 11069, 11104, 11105, 11117, 11119, 13616, 9983,  9982,  15131, 9981,  16094, 16185, 9985,  16656, 9980, 16824, 17485, 17666, 17896, 18244, 18250, 18984, 19018, 9979,  19368, 19476, 21336, 21517, 21518, 22468, 22469, 9896,  19325, 23392, 24905, 24974, 25037, 27010, 27385,  24066, 24067, 24154, 24350, 25519, 9977,  16586, 9976,  26597, 26721, 9984,  6749,  27056, 27065, 27068, 27150, 27183, 27194, 27948, 28057, 28690, 28790, 29250, 29251, 29658, 29740, 29906, 29948, 29959, 29967, 35344, 30039, 30155, 30304, 35290, 35291, 30008, 9988);
UPDATE `creature_template` SET `AIName`='' WHERE `entry`=16764;

DELETE FROM `smart_scripts` WHERE `entryorguid` IN (10052, 9978, 26044) AND `source_type`=0 AND `id`>0;
DELETE FROM `smart_scripts` WHERE `entryorguid` IN (33854) AND `source_type`=0 AND `id`>7;
INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
(10052,0,1,2,62,0,100,0,9820,0,0,0,85,54573,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(10052,0,2,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(10052,0,3,4,62,0,100,0,9820,1,0,0,85,54611,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(10052,0,4,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),   
(10052,0,5,6,62,0,100,0,9820,2,0,0,85,54619,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(10052,0,6,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),    
(10052,0,7,8,62,0,100,0,9820,3,0,0,85,54631,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(10052,0,8,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),  
(10052,0,9,10,62,0,100,0,9820,4,0,0,85,54634,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(10052,0,10,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),     
(10052,0,11,12,62,0,100,0,9820,5,0,0,85,54677,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(10052,0,12,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),   
(10052,0,13,14,62,0,100,0,9820,6,0,0,85,54688,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(10052,0,14,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(9978,0,1,2,62,0,100,0,9820,0,0,0,85,54573,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(9978,0,2,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(9978,0,3,4,62,0,100,0,9820,1,0,0,85,54611,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(9978,0,4,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),   
(9978,0,5,6,62,0,100,0,9820,2,0,0,85,54619,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(9978,0,6,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),    
(9978,0,7,8,62,0,100,0,9820,3,0,0,85,54631,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(9978,0,8,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),  
(9978,0,9,10,62,0,100,0,9820,4,0,0,85,54634,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(9978,0,10,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),     
(9978,0,11,12,62,0,100,0,9820,5,0,0,85,54677,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(9978,0,12,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),   
(9978,0,13,14,62,0,100,0,9820,6,0,0,85,54688,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(9978,0,14,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(26044,0,1,2,62,0,100,0,9820,0,0,0,85,54573,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(26044,0,2,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(26044,0,3,4,62,0,100,0,9820,1,0,0,85,54611,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(26044,0,4,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),   
(26044,0,5,6,62,0,100,0,9820,2,0,0,85,54619,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(26044,0,6,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),    
(26044,0,7,8,62,0,100,0,9820,3,0,0,85,54631,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(26044,0,8,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),  
(26044,0,9,10,62,0,100,0,9820,4,0,0,85,54634,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(26044,0,10,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),     
(26044,0,11,12,62,0,100,0,9820,5,0,0,85,54677,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(26044,0,12,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),   
(26044,0,13,14,62,0,100,0,9820,6,0,0,85,54688,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(26044,0,14,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(33854,0,8,9,62,0,100,0,9820,0,0,0,85,54573,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(33854,0,9,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(33854,0,10,11,62,0,100,0,9820,1,0,0,85,54611,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(33854,0,11,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),   
(33854,0,12,13,62,0,100,0,9820,2,0,0,85,54619,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(33854,0,13,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),    
(33854,0,14,15,62,0,100,0,9820,3,0,0,85,54631,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(33854,0,15,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),  
(33854,0,16,17,62,0,100,0,9820,4,0,0,85,54634,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(33854,0,17,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),     
(33854,0,18,19,62,0,100,0,9820,5,0,0,85,54677,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(33854,0,19,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),   
(33854,0,20,21,62,0,100,0,9820,6,0,0,85,54688,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell"),
(33854,0,21,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,"Stable mster - On gossip select - cast spell");
```

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/15715813-db-npc-fixing-some-stable-masters?utm_campaign=plugin&utm_content=tracker%2F1310&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F1310&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Killyana**
Fix missing gossips and conditions and add non combat pets recovery.
Closes #14720

### CODE REVIEW

File: sql/updates/world/2015_09_13_00_world.sql.sql [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+UPDATE `creature_template` SET `AIName`='' WHERE `entry`=16764;
+
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (10052, 9978, 26044) AND `source_type`=0 AND `id`>0;
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (33854) AND `source_type`=0 AND `id`>7;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
```
Comment (joschiwald) [inline sql/updates/world/2015_09_13_00_world.sql.sql]: can this be handle by the your core script?

Comment (Killyana) [inline sql/updates/world/2015_09_13_00_world.sql.sql]: No there's 4 npcs that uses a SAI.

Comment (joschiwald) [inline sql/updates/world/2015_09_13_00_world.sql.sql]: you could make your core script to use smartai, replace ScriptedAI by SmartAI

File: src/server/scripts/World/npcs_special.cpp [hunk 2]
```diff
@@ -2321,6 +2322,71 @@ class npc_imp_in_a_ball : public CreatureScript
...
+};
+
+class npc_stable_master : public CreatureScript
+{
+public:
+    npc_stable_master() : CreatureScript("npc_stable_master") { }
+    
+    struct npc_stable_masterAI : public SmartAI
+    {
+		npc_stable_masterAI(Creature* creature) : SmartAI(creature) { }
```
Comment (ccrs) [inline src/server/scripts/World/npcs_special.cpp]: tabs?

File: src/server/scripts/World/npcs_special.cpp [hunk 3]
```diff
@@ -2321,6 +2322,71 @@ class npc_imp_in_a_ball : public CreatureScript
...
+    SPELL_CHIKEN = 54677,
+    SPELL_WOLPERTINGER = 54688,
+    STABLE_MASTER_GOSSIP_SUB_MENU = 9820
+};
+
+class npc_stable_master : public CreatureScript
+{
+public:
+    npc_stable_master() : CreatureScript("npc_stable_master") { }
+
```
Comment (ccrs) [inline src/server/scripts/World/npcs_special.cpp]: extra space

File: src/server/scripts/World/npcs_special.cpp (hunk 2 above)
Comment (tkrokli) [inline src/server/scripts/World/npcs_special.cpp]: Correctly observed. Those 2 tabs need to be replaced by 8 spaces.

**Pull request closed without merging**